src/views/v2/rsvp-kitchen-sink/default-full.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Block: RSVP default full
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp-kitchen-sink/default-full.php
+ *
+ * See more documentation about our Blocks Editor templating system.
+ *
+ * @link {INSERT_ARTICLE_LINK_HERE}
+ *
+ * @var Tribe__Tickets__Ticket_Object $rsvp The rsvp ticket object.
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+?>
```

Comment:
:no_entry_sign: There must be exactly one blank line after the file comment

<sub>hash: ba519e466acd01efed87c98152e8df40</sub>